docs/development/includes.py

```diff
@@ -107,9 +107,16 @@ def extract(path, **kwargs):
         c_string_encoding='ascii',
     )
 
-    context = options.create_context()
+    context = Context(
```

Comment:
How do these changes relate to the PR?

Reply:
cython3 removed create_context, not sure when so I looked at the old implementation of create_context and copied it here.

Reply:
Do we have to add this in the dev folder? There no such folder available inside transformers